float32[] data
bool lock_yaw
---
bool rec
